unitTests/ui/CategoryFacet/CategoryFacetBreadcrumbTest.ts

```diff
@@ -58,6 +58,9 @@ export function CategoryFacetBreadcrumbTest() {
       it('build a breadcrumb with the full path if there is no base path', () => {
         const breadcrumb = buildCategoryFacetBreadcrumb();
         const values = $$(breadcrumb).find('.coveo-category-facet-breadcrumb-values');
+        $$(values)
```

Comment:
That's works!
However you can use this workaround (`childNodes[0].nodeValue`) to only get the immediate text value instead:
```
const breadcrumb = buildCategoryFacetBreadcrumb();
const values = $$(breadcrumb).find('.coveo-category-facet-breadcrumb-values');
expect(values.childNodes[0].nodeValue).toEqual('path_one/path_two');
```

Reply:
In some cases, the text was after the <svg>, not always `childNodes[0]`.  So, I wanted to make sure that, in case of future changes that might change the order of element, it would still work

Reply:
Normally the order of the DOM elements should stay the same, it's a good thing if the tests breaks in case there are some changes. For the JSUI, we are careful of not breaking the order of the elements if possible, since some client can manipulate or style it according to their needs. 

Now, for all breadcrumbs, I believe the child node is at position 0. It's different for the `DynamicHierarchicalFacetValues`, in which case the SVG is before the text, which is then at position 1.

<img width="759" alt="Screen Shot 2020-09-15 at 3 56 21 PM" src="https://user-images.githubusercontent.com/4923043/93258515-6a4a3c00-f76c-11ea-9a9f-da69736e58ae.png">

Reply:
Ok. Should I change them all to use `childNodes[0]` instead? That way, we would know the order changed if the tests aren't passing.

Reply:
Yes! You shouldn't modify the element you want to test, before testing it. So it should be childNodes[X] depending on the position of the text.